### DISCUSSION TITLE
Correct invalid gitignored source directory in Babylon Template (version 9.0)

### DISCUSSION THREAD

**Discussion by ivanbrizuela** [primefaces/community#557] [PrimeReact Templates] [answered]
On Babylon Template (version 9.0, last updated on 2023.03.08), this directory is ignored by `.gitignore`: `./public/assets/sass/theme/designer`

I believe this is an error: The content of that directory is required to recompile scss files into proper css, as it is imported by `./public/assets/sass/_theme.scss`, and should be included in the template source code.

Downloading the template and running it directly from the zip file, which does include the `/public/assets/sass/theme/designer` directory and all pre-compiled css files, enables the template to work.

But if you push it to a private git repository to continue working, that ignored directory will not be pushed and any future checkouts - without the `designer` directory - will render the template unusable.

I think this is the same reason why this error was reported: https://forum.primefaces.org/viewtopic.php?t=68679S

**Comment by buraksaglam089** [comment, marked as answer]
Hi, 

Now that the designer file is public, you can remove it from your .gitignore file. Removing it will resolve any issues.

Regards.

**Comment by mertsincan** [comment]
Hi,

Please try the latest version.

Best Regards,
